tests/Bridge/Symfony/Bundle/Command/SwaggerCommandTest.php

```diff
@@ -19,6 +19,7 @@
 
 /**
  * @author Amrouche Hamza <hamza.simperfit@gmail.com>
+ * @group legacy
```

Comment:
I'm also having a lot of this deprecation notices. The library shouldn't trigger it, is there any way to use the new version without BC ?